chia/server/address_manager.py

```diff
@@ -71,6 +73,64 @@ def to_string(self) -> str:
         )
         return out
 
+    def to_bytes(self) -> bytes:
```

Comment:
I think you would likely save quite a lot of time by passing *in* the `bytearray` here, to append to.
This will cause every peer to create a temporary heap allocation, serialize into it and then copy that into the final serialized buffer. Rather than just serializing directly into the final buffer

Reply:
Good suggestion. I've implemented this now and saved a further 0.04 seconds